Artifacts/PgSql/Structure/Admin/0070-Identity-Support.sql

```diff
@@ -0,0 +1,68 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE dbo.aspnetroles ( 
+  "id" varchar(128) NOT NULL,
```

Comment:
should these fields be quoted? by default if we did not quote the column name it will render as lowercase.

Reply:
should we create the same script for sql server too?